OpenGrok

```diff
@@ -18,116 +18,12 @@
-#   - OPENGROK_IGNORE_PATTERNS    Set ignored patterns for indexer to skip.
-#                                   OPENGROK_IGNORE_PATTERNS="-i dummy"
-#                                   OPENGROK_IGNORE_PATTERNS="-i f:dummy"
-#                                   OPENGROK_IGNORE_PATTERNS="-i d:dummy"
-#                                   Multiple entries can be joined together:
-#                                     "-i dummy -i d:tmp -i f:dummy"
-#   - OPENGROK_SCAN_REPOS         Disable Scan for repositories (*)
-#   - OPENGROK_SCAN_DEPTH         how deep should scanning for repos go
-#                                 (by default 3 directories from SRC_ROOT)
-#   - OPENGROK_WEBAPP_CFGADDR     Web app address to send configuration to
-#                                 (use "none" to avoid sending it to web app)
-#   - OPENGROK_WEBAPP_CONTEXT     Context URL of the OpenGrok webapp
-#                                 (by default /source)
-#                                 - FULL reindex is needed once this is used
-#                                 (old already indexed files won't be refreshed)
-#   - OPENGROK_WPREFIX            Disable wildcard prefix search query
-#                                 support (*)
-#   - OPENGROK_TAG                Enable parsing of revision tags into the History
-#                                 view
-#   - OPENGROK_READ_XML_CONFIGURATION file with read only configuration
-#                                 - temporary workaround for bug # 327
-#   - OPENGROK_FLUSH_RAM_BUFFER_SIZE="-m 16" - set RAM buffer size for flushing,
-#                                 default is 16MB per thread, you might try to
-#                                 increase it to 256MB, but do increase JVM to
-#                                 4/8/16GB ! Lucene defaults to 8 threads.
-#                                 Increase JVM memory as noted using JAVA_OPTS
-#   - OPENGROK_PARALLELISM        Default is 0, meaning that parallelism is
-#                                   determined from available processors.
-#   - OPENGROK_LOGGER_CONFIG_PATH Set path to custom logging.properties file.
-#   - OPENGROK_SUBVERSION_USERNAME name of the user that should be used for
-#                                 fetching the history from subversion
-#   - OPENGROK_SUBVERSION_PASSWORD password of the user that should be used for
-#                                 fetching the history from subversion
-#
-#   - OPENGROK_PROFILER           Pause to await user input for profiling.
-#
-# Notes:
-#   (*) Any Non-Empty String will enable these options
+# Copyright (c) 2008, 2017, Oracle and/or its affiliates. All rights reserved.
+# Portions Copyright (c) 2017-2018, Chris Fraire <cfraire@me.com>.
 #
 
 #
-# Copyright (c) 2008, 2017, Oracle and/or its affiliates. All rights reserved.
-# Portions Copyright (c) 2017-2018, Chris Fraire <cfraire@me.com>.
+# OpenGrok Wrapper (initial setup and cron job updating)
```

Comment:
can you add something like "Please refer to the Usage section for supported environment variables." ?

Reply:
OK added